lua/org-roam/config.lua

```diff
@@ -19,50 +19,51 @@ local DEFAULT_CONFIG = {
     ---Bindings associated with org-roam functionality.
     ---@class org-roam.config.Bindings
     bindings = {
+        prefix = "<Leader>n",
```

Comment:
Add a doc comment above explaining what this does.

Reply:
Done.